src/components/Field/SpotifyListeningTo.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useContext } from 'react';
+import { Input, Row, Col, Button, Dropdown, Menu, Form, Switch } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styles from '../../styles/fields.module.scss';
+import { faAlignLeft, faExpandArrowsAlt } from '@fortawesome/free-solid-svg-icons';
+import { faSpotify } from '@fortawesome/free-brands-svg-icons';
+import { FieldProps } from '.';
+import { globalContext } from '../../context/GlobalContextProvider';
+
+export enum STATS_ALIGNMENT {
```

Comment:
Can replace all STATS_ALIGNMENT with ALIGNMENT

Reply:
Solved in the latest commit.